entities/vectorindex/spfresh/config.go

```diff
@@ -19,12 +19,15 @@ import (
 )
 
 const (
-	DefaultMaxPostingSize     = 0 // it means that it will be computed dynamically by the index
-	DefaultMinPostingSize     = 10
-	DefaultReplicas           = 8
-	DefaultRNGFactor          = 10.0
-	DefaultSearchProbe        = 64
-	DefaultCentroidsIndexType = "hnsw"
+	DefaultMaxPostingSize            = 0 // it means that it will be computed dynamically by the index
+	DefaultMinPostingSize            = 10
+	DefaultReplicas                  = 8
+	DefaultRNGFactor                 = 10.0
+	DefaultSearchProbe               = 64
+	DefaultCentroidsIndexType        = "hnsw"
```

Comment:
Exposing default values for internal config in a different package creates friction. It doesn't allow us to freely break internal stuff around without having to "go out" and change things.